backup/queries_statistics.go

```diff
@@ -77,14 +77,16 @@ func GetAttributeStatistics(connectionPool *dbconn.DBConn, tables []Table, proce
 	for _, table := range tables {
 		tablenames = append(tablenames, table.FQN())
 	}
+	// In the query below we need to adjust the `staattnum` as some columns could have been dropped, but during database
+	// restore the tables will be restored from scratch, and their columns will have different attnums.
```

Comment:
How about restoring statistics with the correct column number?

Reply:
to avoid such situation:
1) backup
2) drop/add columns
3) restore

Reply:
> How about restoring statistics with the correct column number?

I did think about the approach with restoring the original table structure and dropping the columns. I see the following disadvantages here:

1. It will affect much more scenarios than the current solution. With the current approach, the backup behavior is changed only for the statistics backup. But the proposed approach will touch the entire database backup/restore and all users (even those not using the statistics backup), making it more susceptible for possible side effects.
2. It will require significantly larger code changes.
 
Both approaches are not perfect, and I've selected the one that seems more safe, isolated and compact.

Reply:
Done